src/components/ui/button/button.scss

```diff
@@ -10,15 +10,15 @@ $secondary-underline-offset-hover: 4px;
 
 .kui-theme--light {
   --color-button--border: #{colors.$black-900};
-  --color-button--border--disabled: #{colors.$grey-400};
+  --color-button--border--disabled: #{colors.$black-900};
   --color-button__text--hover: #{colors.$white-600};
   --color-button__text: #{colors.$black-800};
   --color-secondary-active: #{colors.$white-100};
 }
 
 .kui-theme--dark {
   --color-button--border: #{colors.$white-0};
-  --color-button--border--disabled: #{colors.$grey-900};
+  --color-button--border--disabled: rgb(255 255 255 / 30%);
```

Comment:
You could check for rgb(...) in `_variables.scss` file instead of using rgb(...) color directly.